### DISCUSSION TITLE
New Source: Freshteam

### DISCUSSION THREAD

**Discussion by thomas-vl** [airbytehq/airbyte#33179] [New Connector Request]
## Tell us about the new integration you’d like to have
Which source and which destination? Which frequency?
New Source: Freshteam
https://developers.freshteam.com/api/

## Describe the context around this new integration
Our data team would love to create reports based on this data for our company

## Describe the alternative you are considering or using
CSV exports.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1200367912513076/1200368140634747) by [Unito](https://www.unito.io)

**Comment by po3na4skld** [comment]
## Integration Vetting

**Webhook-based?**
No, just API calls

**Available authentication modes**
Oauth 2.0

**Creating an account**
Account created, credentials in last pass

**How to populate the account with data?**
_Can it be done entirely through the API?_ Yes
_Will it require some work through the UI? If so, what is required?_ Only initial API key retrieval
_Will it require performing some paid action like running an Ad campaign? If it can be done with under $30, do it and **make sure to send the bill to invoices@airbyte.io**. If it costs more than $30 please tag @ johnlafleur_ No
_Do you need any actions from Airbyte to populate the account with data? If so, please tag @ sherifnada for support_ No

**Available streams for sync**

Employees
GET /employees
POST /employees
GET /employees/{id}
PUT /employees/{id}
GET /employee_fields
POST /employee_fields

Branches
GET /branches

Departments
GET /departments

Sub-departments
GET /sub_departments

Business Units
GET /business_units

Teams 
GET /teams

Levels
GET /levels

Timeoffs
GET /time_offs
POST /time_offs
GET /time_off_types
GET /time_off_types/{id}

Roles
GET /roles

Job postings
GET /job_postings
GET /job_postings/{id}
GET /job_posting_fields
GET /job_postings/{id}/applicant_fields

Candidate sources 
GET /candidate_sources
POST /candidate_sources
GET ​/candidate_source_categories

User functions
GET /user_functions

New Hires
POST /new_hires
GET /new_hires/{id}
PUT /new_hires/{id}

**Integration supports incremental sync?**
We can use sort functionality in our requests

List APIs such as the APIs to retrieve all employees’ details return bulk data. In the query string of an API request, you can use a combination of the following parameters to filter and limit the response data

ATTRIBUTE NAME | DATA TYPE | DESCRIPTION
-- | -- | --
page | integer | Page number starting from which the response data is retrieved.Default value: 1Default number of objects in each page: 50
sort_type | string | Specifies whether the list is sorted in ascending or descending order.Valid values: asc, descDefault value: asc
sort | string | Specifies the attribute based on which the data is sorted.

**Other information/blockers**
The permissible number of API calls in a minute is based on the Freshteam plan opted for an organization. The rate limit for each plan is based on the number of subscribed employees in the organization. The rate limit is applicable to an account, irrespective of the IP address from which calls originate.

FRESHTEAM PLAN | RATE LIMIT (PER MINUTE)
-- | --
Sprout | 0
Blossom | Number of subscribed employees
Garden | Number of subscribed employees
Estate | 2 times the number of subscribed employees

[a comment by an automated account is not shown]
